src/main/resources/application.yaml

```diff
@@ -8,12 +8,11 @@ spring:
     username: ${DB_USERNAME:folio_admin}
   jpa:
     open-in-view: true
-    database-platform: org.hibernate.dialect.PostgreSQL10Dialect
     hibernate:
       ddl-auto: none
     properties:
       hibernate:
-        dialect: org.hibernate.dialect.PostgreSQL10Dialect
+        dialect: org.hibernate.dialect.PostgreSQLDialect
```

Comment:
Please remove database-platform and dialect as this is autodetected in this new version of Spring.